docs/src/scs/sbom.md

```diff
@@ -0,0 +1,19 @@
+# Software bill of materials (SBOM)
+Software bill of materials (SBOM) is a formal, machine readable document that provides a list of software components that make up the target software and all its dependencies.
+
+## SBOM usage in Ghaf
+Ghaf framework will use SBOMs for:
+- Vulnerability identification: automatic correlation of SBOM against known vulnerabilities
+- Vulnerability remediation: automatic process to suggest fixes for identified vulnerabilities
+- Dependency analysis: categorization of open-source and closed source software dependencies
```

Comment:
It would be nice to add more information on license tracking. As we will not only use it for FOSS/CS categorization, but rather a bit more advanced license management.

Reply:
> As we will not only use it for FOSS/CS categorization, but rather a bit more advanced license management.

What are the exact plans with this more advanced license management? I don't think I'm aware of that part.

Reply:
Quite simple, actually. Show the exact licenses of the components and warn if some of them are using the licenses that are not suitable for the project (e.g. for legal reasons). 
For example, Dependencytrack is offering policy based license evaluation (I haven't looked into that feature of it, but I assume it is quite similar to BlackDuck for example).
We should study the possibilities and see what we can do. 
If I understand correctly, the CycloneDX should have licenses field per component: 

  "licenses": [{
      "license": {
        "id": "Apache-2.0"
      }
    }],

Which should be enough to cover most of the needs. (and yes, it then depends on the Nix how much info can we get about the dependencies, but it is policy based, so missing license can be a trigger for something as well).

Reply:
Please see https://github.com/tiiuae/ghaf/commit/fe269404240597ce1b0bf4d54e553295669c921a